packages/manager/apps/pci-rancher/src/components/layout-helpers/Layout.component.tsx

```diff
@@ -30,15 +36,12 @@ function RoutingSynchronisation() {
   useEffect(() => {
     // Need to also hide the preloader here due to firefox still display it when cache is disabled
     // Need to investigate why preloader is not hidden
-    shell.ux.hidePreloader();
-  }, []);
+    trackCurrentPage();
```

Comment:
this will trigger trackPage event automatically when changing page so no need to trackCurrentPage on every page